Tools/CISupport/ews-build/steps.py

```diff
@@ -193,6 +195,34 @@ def fetch_data_from_url_with_authentication_github(self, url):
         else:
             defer.returnValue(response)
 
+    @defer.inlineCallbacks
```

Comment:
@JonWBedard Tried implementing twistd requests here and moved the function to GitHubMixin as requested!